pyvista/core/_validation/check.py

```diff
@@ -921,7 +1175,7 @@ def check_iterable_items(
         raise
 
 
-def check_contains(*, item, container, name='Input'):
+def check_contains(*, item: Any, container: Any, name: str = 'Input'):
```

Comment:
I'm not happy with this API, forcing all kwargs is awkward and inconsistent with the rest of the package.

consider changing to:
`check_contains(container: Any, *, contains: Any, name: str = 'Input')`

Reply:
> consider changing to: `check_contains(container: Any, *, contains: Any, name: str = 'Input')`

This order (container, item) is consistent with python's [operator.contains](https://docs.python.org/3/library/operator.html#operator.__contains__) function API.

The API has been updated to `check_contains(container: Any, /, *, must_contain: Any, name: str = 'Input')` in 9ca27dfab49c230e665fc2a4318408a0fa73acf1